docs/source/training_pipeline/01_getting_started.rst

```diff
@@ -0,0 +1,283 @@
+         :linenos:
+
+         # Create conda environment & activate
+         conda create -n pkd-training python=3.8
+         conda activate pkd-training
+
+         # Clone PeekingDuck repository
+         git clone <TODO: PeekdingDuck repository>
+         cd PeekdingDuck
+
+         # Install required packages
+         pip install -r peekingduck/training/requirements.txt
+
+      **CUDA GPU**
+
+      .. code-block:: bash
+         :linenos:
+
+         # Create conda environment & activate
+         conda create -n pkd-training python=3.8
+         conda activate pkd-training
+
+         # Clone PeekingDuck repository
+         git clone <TODO: PeekdingDuck repository>
+         cd PeekdingDuck
+
+         # Install required packages
+         pip install -r peekingduck/training/requirements.txt
+
+         # Install cuda dependencies
+         conda install -c conda-forge cudatoolkit=11.2.2 cudnn=8.1.0
+
+         # Initialize cuda
+         mkdir -p $CONDA_PREFIX/etc/conda/activate.d \
+         && mkdir -p $CONDA_PREFIX/etc/conda/deactivate.d \
+         && echo 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CONDA_PREFIX/lib/'>\
+         $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh && \
+         echo 'unset LD_LIBRARY_PATH'>\
+         $CONDA_PREFIX/etc/conda/deactivate.d/env_vars.sh
+
+         # Reinitialize environment
+         conda activate base && conda activate pkd-training
+
+   .. tab:: Mac 
+    
+      Platform:
+         - MacOS Intel Chip or MacOS M1/M2 chip
+      Prerequisite:
+         - `Git <https://github.com/git-guides/install-git>`_
+         - `Miniforge <https://github.com/conda-forge/miniforge#homebrew>`_
+
+         On macOS, you can install miniforge with Homebrew by running
+
+         .. code-block:: bash
+            :linenos:
+
+            brew install miniforge
+
+      Install requirements
+
+         .. code-block:: bash
+            :linenos:
+
+            # Clone PeekingDuck repository
+            git clone <TODO: PeekdingDuck repository>
+            cd PeekdingDuck
+
+            # Create conda environment & activate
+            conda env create -f peekingduck/training/requirements_mac.yaml
+            conda activate pkd-training
+
+      Install TensorFlow
+
+         .. code-block:: bash
+            :linenos:
+            
+            bash .peekingduck/training/scripts/install_tensorflow_macos.sh
+
+.. _setting-up-weights-and-biases:
+
+Setting up Weights & Biases
+---------------------------
+
+Follow the steps below to get configure weights & biases:
+
+1. `Sign up <https://wandb.ai/site>`_ for a free account and then login to your wandb account.
+2. (If not yet installed via requirements.txt) Pip install the wandb library on your machine in a Python 3 environment.
+3. Login to the wandb library on your machine. You will find your API key `here <https://wandb.ai/authorize>`_
+
+.. code-block:: bash
+   :linenos:
+   
+   pip install wandb
+
+   # Login to weights & biases
+   wandb login
```

Comment:
For above `code-block` (and all other related portions below and elsewhere), our convention for showing screen input/output is to use `.. admonition:: Terminal Session` together with the actual color coded display. You can reference the sample in https://raw.githubusercontent.com/aisingapore/PeekingDuck/main/docs/source/tutorials/01_hello_cv.rst

The terminal display should so both the user input and the expected output.
And to be instructive and complete, you will want to show the prompt where the user inputs the PAT into W&B, because this is the last step.